terraform/050-pw-manager/main-ui.tf

```diff
@@ -96,7 +96,7 @@ resource "aws_iam_user_policy" "ci_ui" {
         "cloudfront:CreateInvalidation"
       ],
       "Resource": [
-        aws_cloudfront_distribution.ui[0].arn
+        "${aws_cloudfront_distribution.ui[0].arn}"
```

Comment:
is this right? looks like 0.11 syntax

Reply:
I'm pretty sure that the index `[0]` means you need the old syntax.